bin/metabuild_common/entrypoint.clj

```diff
@@ -0,0 +1,20 @@
+(ns metabuild-common.entrypoint
+  (:require [clojure.pprint :as pprint]
+            [colorize.core :as colorize]))
+
+(defn do-exit-when-finished-nonzero-on-exception [thunk]
+  (try
+    (thunk)
+    (System/exit 0)
+    (catch Throwable e
+      (let [e-map (Throwable->map e)]
+        (println (colorize/red (str "Command failed: " (:cause e-map))))
+        (binding [pprint/*print-right-margin* 120]
+          (pprint/pprint e-map))))))
```

Comment:
Aren't you missing a `(System/exit 1)` in here, as the name suggests?

Reply:
Yeah. I fixed this in #13417